### DISCUSSION TITLE
Ability to generate plain text

### DISCUSSION THREAD

**Discussion by eric-mathison** [apostrophecms/sanitize-html#629] [Q&A]
Can I use this to convert html to plain text? Can this be done by simply disallowing all tags? 

This library throws errors when html-to-text is added as well.

**Comment by BoDonkey** [comment]
@eric-mathison I'm not sure - you would have to make sure the settings were correct to remove just the tags, and not all of the content in the tags. Can you provide instructions to replicate the errors that you are getting?

**Reply by eric-mathison** [reply]
I just tried it again today and of course, I'm not getting the error now. 
Looking back in the logs this was the error: 
- error ./node_modules/html-to-text/lib/html-to-text.mjs
Attempted import error: 'render' is not exported from 'dom-serializer' (imported as 'render').

The error would fire on first render after adding this library. But as I mentioned, it seems to be working now.
